Task150/Task150/Task150/Pages/WomenTabPage.cs

```diff
@@ -0,0 +1,68 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Interactions;
+using SeleniumExtras.PageObjects;
+using Task150.HelperPage;
+
+namespace Task150.Pages
+{
+    internal class WomenTabPage : Page
+    {
+        [FindsBy(How = How.XPath, Using = "//a[contains(@class,'button lnk_view')]")]
+        private IWebElement MoreButton;
+
+        public WomenTabPage(IWebDriver driver) : base(driver)
+        {
+            PageFactory.InitElements(driver, this);
+        }
+
+        public bool IsThisPage()
+        {
+            return _driver.Title.Equals("Women - My Store");
+        }
+
+        public ProductPage ChooseProduct()
+        {
+            ActionMouseToProduct(1);
+            MoreButton.Click();
+            return new ProductPage(_driver);
+        }
+
+        public WomenTabPage AddedThreeProductToCart()
+        {
+            for (var i = 1; i <= 3 && IsThisPage(); i++)
+            {
+                ActionMouseToProduct(i);
+                var byAddToCartButton = "//a[contains(@class, 'ajax_add_to_cart_button')]";
+
+                if (_driver.WaiterByElementIsDisplay(By.XPath(byAddToCartButton)))
+                {
+                    var byaddToCartButton = By.XPath(GetProductStringXpathByNumber(i) + byAddToCartButton);
+                    if (_driver.WaiterByElementIsDisplay(byaddToCartButton))
+                    {
+                        _driver.FindElement(byaddToCartButton).Click();
+                    }
+
+                }
```

Comment:
I don't think that this method will be able used with expecting flexibility. Try to create a general method AddItem() and then call it inside method like AddItems(int amountItems) { foreach(amountItems) {AddItem()}